2_Growth-Maturity-Decline.md

```diff
@@ -1,10 +1,10 @@
 # Category: Growth-Maturity-Decline
 
-Goal: Describe category and the overall goal of the category
+Goal: Identify if the activity of an open source community is growing or declining and the level of maturity the project has reached.

```

Comment:
Thanks @BenLloydPearson 

How about changing the order: Identify the level of maturity the open source project has reached and whether its community is growing or declining.

Reply:
I agree, I've changed it.